examples/compiled/box-plot_minmax_2D_horizontal_IQR.vg.json

```diff
@@ -21,17 +21,23 @@
                     "ops": [
                         "q1",
                         "q3",
-                        "median"
+                        "median",
+                        "min",
+                        "max"
                     ],
                     "fields": [
+                        "people",
+                        "people",
                         "people",
                         "people",
                         "people"
                     ],
                     "as": [
                         "lowerBox",
                         "upperBox",
```

Comment:
Can we name it upper/lower/midBox_fieldName to avoid name collision?

Reply:
@kanitw should IQR should have fieldname appended? So, it would be "IQR_people".